client/css/style.css

```diff
@@ -915,6 +915,13 @@ button, .user {
 	padding-top: 2px;
 	line-height: 1.8;
 }
+#settings #change-password .error {
+	color: #e74c3c;
+	margin-bottom: 1em;
+}
+#settings #change-password .error.success {
+	color: #2ecc40;
```

Comment:
to the first point, why a margin-bottom on .error but not .error.success: the second rule will inherit from the first so there is no reason to override the margin-bottom with an identical value when it is already applied by inheritance.

.error.success: that's a semantic issue I guess, but we can rename .error if need be.